macros_doc.md

```diff
@@ -9,15 +34,40 @@ such as Excel.
 
 MkDocs (the tool we use to turn the markdown version of the specification into
 HTML pages) supports macros. In the BIDS specification document, we use these
-macros to standarize the format of items such as tables and examples.
+macros to standardize the format of items such as tables and examples.
+
+## What kind of input information are required by macros?
+
+It will depend for each macro.
+
+For some of them, all the inputs will be directly available in the markdown
+document where you have put the call to the macro.
+
+Other macros will need to be pointed to some external input (like a metadata
```

Comment:
Replace the first sentence in lines 46-47 with:

Other macros use information (such as metadata terms) from external sources, and you will need to provide links to this information as part of the call.